presto-main/src/main/java/com/facebook/presto/connector/informationSchema/InformationSchemaMetadata.java

```diff
@@ -62,7 +62,6 @@
                     .column("column_default", VARCHAR)
                     .column("is_nullable", VARCHAR)
```

Comment:
It removes it from `SHOW COLUMS`. This is a standard field in `information_schema.columns`.

The non-standard field should never have been added (to that table).